advanced_source/super_resolution_with_onnxruntime.py

```diff
@@ -301,8 +327,12 @@ def to_numpy(tensor):
 # Save the image, we will compare this with the output image from mobile device
 final_img.save("./_static/img/cat_superres_with_ort.jpg")
 
+# Save resized original image (without super-resolution)
+img = transforms.Resize([img_out_y.size[0], img_out_y.size[1]])(img)
+img.save("cat_resized.jpg")
 
 ######################################################################
+# .. figure:: /_static/img/cat_resized.jpg
```

Comment:
So now we have this image displayed twice: https://docs-preview.pytorch.org/pytorch/tutorials/2935/advanced/super_resolution_with_onnxruntime.html 

If you are comparing them, add some text in between the two images?

Reply:
I've added captions to each of the two images.